webapp/reviews/models.py

```diff
@@ -14,13 +15,7 @@ class Review(models.Model):
         'Rating',
         null=True,
         blank=True,
```

Comment:
This is a field that shouldn't be allowed to be blank
I can link you the Django docs on it, but the skinny is that Blank fields only apply to text fields; for consistency in DB queries and such, Null values should be used for non-strings.

Reply:
Can you show me the docs? This is what I'm basing my knowledge off of: https://docs.djangoproject.com/en/1.11/ref/models/fields/#blank

My line of thinking is that a user might be able to submit a comment without an attached rating. 

`null=True` means that the database can hold a null value for `rating`.
`blank=True` means that the user can actually *input* a null value for `rating`, if I'm interpreting the docs correctly.

Reply:
Oh snap; guess I had misunderstood. Might've confused this with something else.
Never mind this change 👍